tutorials/graph-hello-world/graph-hello-world.md

```diff
@@ -0,0 +1,169 @@
+            console.log(url) //for debugging
+            const options = {
+                method: "get",
+                headers:{
+                    "Accept": "application/json",
+                    "apiKey": "your APIkey"
+                }
+            };
+            const response = await fetch(url, options);
+            console.log(`${response.status} (${response.statusText})`) // for debugging
+            const json = await response.json();
+            return json;
+        }
+    }
+    module.exports = Graph;
+    ```
+
+    > Note that the SAP Graph server that uses SAP API Business Hub sandbox is hard-coded. This allows you to focus here on the data access API, without requiring you to first configure your own SAP Graph business data graph and deal with all the complex aspects of security and authentication, which will be the subject of future tutorial parts.
+    Since you are accessing the sandbox landscape data via SAP API Business Hub, you will need to insert your API Key (a short string) into the following code. Where do you get this key?
+
+2. Go and login to <https://api.sap.com/settings> and choose **Show API Key** to see and save it.
+
+[DONE]
+[ACCORDION-END]
+---
+[ACCORDION-BEGIN [Step 5: ](Write a simple application)]
+
+Now you are ready to write your first, simple server-side application that uses SAP Graph.
+
+1. Paste the following into a new file, called `hellograph.js`, in the `src` folder:
+
+    ```JSON
+    const express = require("express");
+    const app = express();
+    const Graph = require("./graph");
+    const port = 3004;
+    const graph = new Graph();
+    app.get('/sap*', async (req, res) => {
+        const response = await graph.get(req, `${req.url}`, "");
+        res.send(`<pre><code>${JSON.stringify(response, null, 2)}</code></pre>`);
+    });
+    app.listen(port, () => { console.log(`Explore SAP Graph at http://localhost:${port}`) });
+    ```
+
+    As mentioned, this code uses a popular Node.js package called **Express**. On line 8, you define the action taken when your server-side app is called from a browser. Here, you simply take the URL that you received (`req.url`), and pass it through, as-is, to SAP Graph. The returned data from SAP Graph is then shown as a raw result in the browser screen.
+
+2. Your server-side app is ready. To run it, use your terminal console to enter:
+
+    ```Shell / Bash
+    node hellograph.js
+    ```
+
+The console will show:
+> _Explore SAP Graph at <http://localhost:3004>_
+Now the fun begins. The application expects a well-formed query URL to function properly.
+
+3. Open a browser window or tab, and enter the following query URL:
+`http://localhost:3004/sap.graph/SalesQuote?$top=2`
+
+The browser will invoke your application code, which will call SAP Graph to fetch the data, and if all went well, you will see the following output on your screen:
+
+!![Output](blog2_output_sm.png)
```

Comment:
If it's not too much trouble, please remove that big, white box around the actual image.